src/components/SkeletonPage/SkeletonPage.scss

```diff
@@ -32,6 +32,43 @@ $action-text-spacing: rem(24px);
   margin-bottom: (-1 * spacing(extra-tight));
 }
 
+.TitleAndPrimaryAction {
+  display: flex;
+
+  @include page-content-when-partially-condensed {
+    display: block;
+  }
+}
+
+.Title {
+  flex: 1 1 0%;
+}
+
+.PrimaryAction {
+  align-self: stretch;
+  align-items: center;
```

Comment:
I don't fully understand these.

Isn't `align-items` placed on the parent class that contains `display: flex` and
 `align-self` placed on a child class? I don't understand the presence of both of these on the same class.